docs/v0.1/1001-install.md

```diff
@@ -2,12 +2,6 @@
 slug: /1001/install/
 ---
 
-<!--
```

Comment:
Note that I have removed this block, originally added by @slumbering, because it seems to disrupt with the auto-generated titles in the sidebar navigation.